df.world-site.xml

```diff
@@ -431,14 +431,16 @@
             <static-array count='51' pointer-type='site_realization_crossroads'/>
         </static-array>
         <static-array name='river_map' count='51'>
-            <pointer>
-                <static-array name='start' type-name='int32_t' count='48'/>
-                <static-array name='end'   type-name='int32_t' count='48'/>
-                <int32_t name="unk_180"/>
-                <int32_t name="unk_184"/>
-                <int32_t name="start_array_limits"/>
-                <int32_t name="end_array_limits"/>
-            </pointer>
+            <static-array count='51'> 
```

Comment:
world_site is 816 x 816 pixels
816/51 = 16 so each entry in the map corresponds to a 16x16 area.
start is a 48 int array = 48 / 3 local pixel per pixel embark = 16, which is the previous area.

I don't have time now but tomorrow I'll submit some images to prove it